project/Build.scala

```diff
@@ -709,6 +709,7 @@ object Build {
       mimaBinaryIssueFilters ++= BinaryIncompatibilities.LinkerInterface,
       exportJars := true, // required so ScalaDoc linking works
 
+      libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % "test",
```

Comment:
That seems wrong. This is a JVM dependency but it goes into the common settings.

Reply:
Fixed in b58a8673c6b5bd196a2e9af6d86353b59a91ec12. Is this the correct way to do it?